terrainbento/base_class/erosion_model.py

```diff
@@ -606,9 +591,9 @@ def run(self):
             )
             self.run_for(self.clock.step, next_run_pause - time_now)
             time_now = self._model_time
-            self.iteration += 1
             self._itters.append(self.iteration)
             self.write_output()
+            self.iteration += 1
 
```

Comment:
FYI @gregtucker this is where there was a bug. 

Specifically, this meant that when IC is written the time coordinate of the netcdf was written as 

(0, 2, 3, 4, 5 ... nts) x dt 

instead of 

(0, 1, 2, 3, 4, ... nts) x dt

Fixing this meant that the "truth" netcdfs time array was wrong.